alphalens/tears.py

```diff
@@ -417,6 +419,24 @@ def create_returns_tear_sheet(factor_data,
         plt.show()
         gf.close()
 
+    if factor_returns.index.tzinfo is None:
```

Comment:
If they're different, it's not as big of a deal, no? pandas can handle differing time zones, so the `intersection` that occurs later is still safe. If returns are in `EST` and the factors are in `UTC`, then as long as the datetimes still match up, e.g. 9AM UTC and 4AM EST, it will work. The issue is whether the `factor_returns` are totally naive, which is what we want to check for.

Reply:
Though a check later to make sure that they DO match up, like the market open issue that you mentioned above, would be good.